.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "third_party/cccl"]
 	path = third_party/cccl
-	url = https://github.com/NVIDIA/cccl.git
+	url = https://github.com/cupy/cccl.git
```

Comment:
So is the plan to switch to a CuPy friendly fork of CCCL for now or is this just until the changes we need land?